### PR TITLE
[WIP] Multi-obligation envelopes

### PR DESCRIPTION
**Pull request by andrei-duhnea**

### CODE REVIEW

File: reportek/core/api/views/reporting.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import logging
 from base64 import b64encode
 from django.views import static
-from django.db.models import Q, F, Exists, OuterRef
```
Comment (sorinstelian) [inline reportek/core/api/views/reporting.py:8]: FilteredRelation not used in the end. :)

Comment (andrei-duhnea) [reply reportek/core/api/views/reporting.py:8]: Indeed, it was weak sauce :)

File: reportek/core/serializers.py [hunk 2]
```diff
@@ -430,25 +438,91 @@ class Meta(FeedbackCommentSerializer.Meta):
...
+            read_only=True,
+            fields=('id', 'title', 'terminated')
+        ).data
+
+    class Meta:
+        model = EnvelopeObligationSpec
+        fields = ('obligation_spec', 'obligation', 'reporting_cycle')
+
+
+class NestedEnvelopeObligationSpecSerializer(
```
Comment (sorinstelian) [inline reportek/core/serializers.py]: Is this still needed?

Comment (andrei-duhnea) [reply reportek/core/serializers.py]: No, it will be going away with the next push. Thanks for the catch.

**Pull request merged**